src/Microsoft.DotNet.Cli.Utils/ProjectContextExtensions.cs

```diff
@@ -63,5 +69,118 @@ public static string GetDefaultRootOutputPath(ProjectContext context, string cur
 
             return rootOutputPath;
         }
+        
+        internal static void MakeRunnable(this ProjectContext context, string outputPath, string configuration)
+        {
+            context
+                .ProjectFile
+                .Files
+                .GetCopyToOutputFiles()
+                .StructuredCopyTo(context.ProjectDirectory, outputPath)
+                .RemoveAttribute(FileAttributes.ReadOnly);
```

Comment:
Agreed this is pre-existing behavior, but it looks to have come from the function `CopyContents` which offered some documentation as to its function. 
Would you consider adding a comment that this is intending to copy contents?